hack/scale_out_poc/haproxy_2T1R/haproxy.cfg

```diff
@@ -74,14 +43,13 @@ defaults
         errorfile 503 /etc/haproxy/errors/503.http
         errorfile 504 /etc/haproxy/errors/504.http
 
-# the following setup enable the prometheus export
-frontend stats
-    bind *:8404
-    option http-use-htx
-    http-request use-service prometheus-exporter if { path /metrics }
-    stats enable
-    stats uri /stats
-    stats refresh 10s
+KUBEMARK_ONLY:frontend stats
```

Comment:
I will later have a change to enable this even for one-box environment.

Reply:
here actually https://github.com/pdgetrf/arktos/commit/9956e2c7a00abad918745da5bb875eaaf8cb782e

but okay to check in with your change for now.